database/layer/knex.js

```diff
@@ -16,15 +16,12 @@ try {
   usherDb = knex(knexDbConfig)
   if (usherDb?.client?.pool) {
     const pool = usherDb.client.pool
-    // Set idle timeout to 0 to release connections immediately. This can't be configured through Knex.
+    /* 
+      Set idleTimeoutMillis to 0 to ensure connections are released during the next pool.check() interval.
+      The check interval can be configured using reapIntervalMillis.
+      Default idleTimeoutMillis is 30000 ms.
+    */
     pool.idleTimeoutMillis = 0
-
-    // Check the pool for idle connections on 'release' event
-    pool.on('release', () => {
```

Comment:
@k1ch  was this change not already released?

Reply:
@mikelax Yes, it was released in [v2.2.1](https://github.com/DMGT-TECH/the-usher-server/releases/tag/v2.2.1). However, I decided to remove it and instead rely on the [tran](https://www.npmjs.com/package/tarn) interval, which can be configured using the `reapIntervalMillis` attribute, instead of using `process.nextTick()`. The latter supersedes all processes and might cause race conditions.